lib/modules/manager/bitbucket-pipelines/readme.md

```diff
@@ -1,3 +1,50 @@
 Extracts dependencies from Bitbucket Pipelines config files.
 
 If you need to change the versioning format, read the [versioning](https://docs.renovatebot.com/modules/versioning/) documentation to learn more.
+
+### Using private build images
+
+Bitbucket Pipelines supports using an authenticated Docker registry, including self-hosted private registries, Docker Hub, Amazon ECR and Google GCR.
+
+More details can be found on the Atlassian [Bitbucket Pipeline documentation](https://support.atlassian.com/bitbucket-cloud/docs/use-docker-images-as-build-environments) site
+
+The following `regexManager` configuration can be used for authenticated Docker Hub or private registries. Changes will be required for Amazon ECR or Google GCR use cases.
```

Comment:
```suggestion
The following `regexManager` configuration can be used for authenticated Docker Hub or private registries.
You'll need to change this example config if you're using Amazon ECR or Google GCR.
```

Also: do our readers know _what_ changes they'll need to make if they're using Amazon ECR or Google GCR? I don't see this explained in the text, but maybe we can assume our readers are competent with ECR/GCR already?